go/libraries/doltcore/table/typed/parquet/writer_test.go

```diff
@@ -83,12 +82,7 @@ John Johnson,21,
 Andy Anderson,27,
 `
 
-	file, err := ioutil.TempFile("", "parquet")
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer os.Remove(file.Name())
```

Comment:
Looks like you dropped this defer block

Reply:
Actually, seems like the testing framework will remove this for us on test cleanup, so no need for this afterall, nevermind